### ISSUE TITLE
assert should return true values

### ISSUE THREAD

**Issue by weinholt** [IronScheme/IronScheme#75]
This part of `assert` in R6RS was not implemented in psyntax from r6rs-libraries:

> If expression returns a true value, that value is returned from the assert expression.

Example:

```
IronScheme 1.0.239-671ea21 github.com/IronScheme © 2007-2019 Llewellyn Pritchard (.NET Core 3.0 64-bit)
> (assert #t)
> 
```

Compare with Chez:

```
Chez Scheme Version 9.5.4
Copyright 1984-2020 Cisco Systems, Inc.

> (assert #t)
#t
>
```

**Comment by weinholt** [comment]
Oops! I forgot to mention you, @leppie.

**Comment by leppie** [comment]
Should be fixed with 687cde5 . Release is up.
